components/ItemCard.tsx

```diff
@@ -21,11 +22,20 @@ const ItemCard: React.FC<ItemCardPropType> = ({
   postcode,
   postable,
   itemId,
+  reserved,
 }) => {
+  console.log('Reserved:', reserved);
   return (
     <div className='card'>
       <Link href={`/item/${itemId}`} className=''>
-        <h2 className='p-4  font-semibold'>{item_name}</h2>
+        <div className='flex flex-row justify-between align-middle'>
+          <h2 className='p-4  font-semibold'>{item_name}</h2>
+          {reserved && (
+            <span className='place-self-center rounded-full bg-primaryOrange p-2'>
+              Reserved
+            </span>
```

Comment:
If we're doing it twice already what about just packaging it up into a class in our `globals.css`?

With our new milestone for browsing I can imagine we'll rework and reuse this little flag a few times an having it neatly referenced in one spot will make it easier.

Reply:
Should I make the whole **Reserved** label as a component itself? Or you prefer to have just the classes?

Reply:
You know what, I spent a little bit thinking about this as well. I was thinking making a component out of it in case we want to show it in the `CurrentConversation` for example, but I think the application might change enough that we'd not really re-use the same component and since it's not doing any data displaying or anything it's probably enough to have it as a class we can apply to any `<p>` at will.

If you think there's a case for a full component I won't oppose it either, but it's probably a justifiable iteration right now to stick to a css class